scripts/monorepo/src/getDependencies.js

```diff
@@ -2,6 +2,10 @@ const fs = require('node:fs');
 
 const { createProjectGraphAsync, joinPathFragments, workspaceRoot } = require('@nx/devkit');
 
+/** @import { PackageJson } from './types';
+ *  @import { ProjectGraph } from '@nx/devkit';
+ */
+
```

Comment:
this might be invalid JSDoc

```suggestion
/** 
 *  @import { PackageJson } from './types';
 *  @import { ProjectGraph } from '@nx/devkit';
 */

```